MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1148 @@
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in .NET MAUI DataForm (SfDataForm)
+
+The data form supports several built-in editors as follows:
+
+<table>
+<tr>
+<th>Editor name</th>
+<th>Supported Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+The string type property and any other type apart from the below specified cases.
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+MultilineText
+</td>
+<td>
+The string type property with multi line text.
+[DataType(DataType.MultilineText)] attribute.
+</td>
+<td>
+{{'[Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Password
+</td>
+<td>
+The string type property with 
+[DataType(DataType.Password)] attribute.
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Date
+</td>
+<td>
+DateTime, DateOnly or DateTimeOffset type property or the property with [DataType(DataType.Date)] or [DataType(DataType.DateTime)] attributes..
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Time
+</td>
+<td>
+The TimeSpan and TimeOnly type property or the property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+Enum and List type property. 
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+</table>
+
+## To change the editor for any data type
```

Comment:
## Change the editor for any data type

By default, the editors will be loaded based on the based on the primitive data type such as string, enumeration, DateTime, and TimeSpan

Reply:
title and definition changed